client/settings/test/settings-section.tsx

```diff
@@ -33,7 +33,7 @@ describe( 'SettingsSection', () => {
 
 	test( 'renders children', () => {
 		render(
-			<SettingsSection title="Foo" description="Bar">
+			<SettingsSection title="Foo">
```

Comment:
Why do we drop `description` here?

Reply:
I think this was a mistake. Originally the `SettingsSection` component did not have a `description` property. In my understanding, this was not making any effect.

After this PR, I refactored the original name of the property `Description` to `description`, to comply with the TS linter naming-conventions rule. And now the type of description is `React.FunctionComponent`, so setting it to `string` is not allowed. It is worth mentioning that there were no cases were the `Description` was passed as `string`.

Hopes this makes sense but let me know otherwise